test/runtests.jl

```diff
@@ -61,6 +61,21 @@ cd(dirname(@__DIR__)) do
     open("fakefile",true,true,true,false,false)
     @test isempty(Coverage.process_cov("fakefile",datadir))
     rm("fakefile")
+
+    # Test `using Coverage` with non-empty command-line arguments
+    script = tempname()
```

Comment:
some operating systems clean up temporary directories automatically, but better not to have to rely on that